### DISCUSSION TITLE
How to find the sources of the largeboard version

### DISCUSSION THREAD

**Discussion by 38alysia** [fairy-stockfish/Fairy-Stockfish#793] [Q&A] [answered]
the releases below Fairy-Stockfish 14.0.1 XQ including largeboard x86-64 , however when i download the source code zip and then makefile  and open it by using `./stockfish load variants.ini` , it happens that 
**Fairy-Stockfish 14 by Fabian Fichter
Variant template 'xiangqi' does not exist.
Variant template 'janggi' does not exist.
Variant template 'janus' does not exist.
Variant template 'grand' does not exist.
Variant template 'capablanca' does not exist.**
Does the zip below  Fairy-Stockfish 14.0.1 XQ supports largeboard?

**Comment by 38alysia** [comment]
When making file , the output log is as following:
**lto-wrapper: warning: jobserver is not available: ‘--jobserver-auth=’ is not present in ‘MAKEFLAGS’
lto-wrapper: warning: using serial compilation of 15 LTRANS jobs**

**Comment by ianfab** [comment, marked as answer]
https://github.com/fairy-stockfish/Fairy-Stockfish/wiki/Compiling-Fairy-Stockfish#compiling-version-for-large-boards-8x8

**Reply by 38alysia** [reply]
I get it，thanks！
